### PR TITLE
Feature/logout functionality

### PR DESCRIPTION
**Pull request by HarisShahidHameed**
Trello link - https://trello.com/c/mS35JpXw/17-logout-functionality

### CODE REVIEW

File: server/libs/userlibs.js [hunk 1]
```diff
@@ -108,4 +119,33 @@ const updateUser = function (body) {
...
+                status : statusCodes.badRequest,
+                message: `${messages.wrong}`   
+            })
+        }).catch((error) => {
+            reject({error});
+        })
+    })
+}
+
+module.exports = {register, login, updateUser, logOut}
```
Comment (abdulhaqshah) [inline server/libs/userlibs.js]: login should be logIn

File: server/libs/userlibs.js [hunk 2]
```diff
@@ -108,4 +119,33 @@ const updateUser = function (body) {
...
 };
 
-module.exports = {register, login, updateUser}
\ No newline at end of file
+const logOut = function(request) {
+    return new Promise((resolve,reject) => {
+        let token = request.header('x-authentication');
+        let decoded = {};
+        try {
+            decoded = jwt.verify(token, secretKeys.tokenKey, process.env.JWT_SECRET);
```
Comment (abdulhaqshah) [inline server/libs/userlibs.js:127]: why we are verifying token in libs level

File: tests/seed.js [hunk 3]
```diff
@@ -24,4 +26,24 @@ const populateUsers = done => { //eslint-disable-line
     }).then(() => done()); //eslint-disable-line
 };
 
-module.exports = {users, populateUsers};
\ No newline at end of file
+const setUuid = (index) => {
+    return new Promise((resolve,reject) => {
```
Comment (abdulhaqshah) [inline tests/seed.js]: remove this function and get uuid from the created user

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]

File: tests/seed.js (hunk 3 above)
Comment (HarisShahidHameed) [reply tests/seed.js]: Removed

File: server/libs/userlibs.js (hunk 2 above)
Comment (HarisShahidHameed) [reply server/libs/userlibs.js:127]: We are not verifying token here. We are taking time at which this token was created from it's payload

File: server/libs/userlibs.js (hunk 1 above)
Comment (HarisShahidHameed) [reply server/libs/userlibs.js]: Done

**Pull request merged**